examples/petstore-expanded/api/petstore-client.gen.go

```diff
@@ -124,44 +124,44 @@ func NewClientWithResponses(server string) *ClientWithResponses {
 	}
 }
 
-// FindPetsResponse is returned by Client.FindPets()
-type FindPetsResponse struct {
+// OAPIFindPetsResponse is returned by Client.FindPets()
+type OAPIFindPetsResponse struct {
```

Comment:
Things like this is why I was avoiding generating responses :) 

I think that name collisions are a valid issue with these new changes, but I also think that 'OAPIFundPetsResponse' is kinda gross.

The user isn't going to ever be instantiating these types, how about we do away with type names altogether? Declare your functions to return `struct {whatever}`. The struct will be declared in two places - the return type, and at time of creation of the local variable into which to unmarshal.

Reply:
I agree that it is kinda gross, but the anonymous structs are not going to look pretty either. Also how would we deal with the `StatusCode()` and `Status()` methods?

What if we keep the response structs with their previous names, but make them private? That way they'll never clash with the generated schema models.

Does that work?